src/components_new/SpellCheckSidebar.tsx

```
@@ -0,0 +1,423 @@
+
+  if (!isOpen) return null;
+
+  return (
+    <AnimatePresence>
+      <motion.div
+        {...({
+          initial: { x: "100%" },
+          animate: { x: 0 },
+          exit: { x: "100%" },
+          transition: { duration: 0.3, ease: "easeInOut" }
+        } as any)}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)